src/main/java/se/sundsvall/billingpreprocessor/api/model/Recipient.java

```diff
@@ -2,37 +2,42 @@
 
 import io.swagger.v3.oas.annotations.media.Schema;
 import jakarta.validation.Valid;
+
+import se.sundsvall.dept44.common.validators.annotation.ValidOrganizationNumber;
 import se.sundsvall.dept44.common.validators.annotation.ValidUuid;
 
 import java.util.Objects;
 
 import static io.swagger.v3.oas.annotations.media.Schema.RequiredMode.REQUIRED;
 
-@Schema(description = "Billing issuer model")
-public class Issuer {
+@Schema(description = "Billing recipient model")
+public class Recipient {
 
-	@Schema(description = "Unique id for the person issuing the billing record. Mandatory for EXTERNAL billing record.", example = "f0882f1d-06bc-47fd-b017-1d8307f5ce95")
+	@Schema(description = "Unique id for the person issuing the billing record. Mandatory for EXTERNAL billing record if organizationNumber is null.", example = "f0882f1d-06bc-47fd-b017-1d8307f5ce95")
 	@ValidUuid(nullable = true)
 	private String partyId;
+	@Schema(description = "OrganizationNumber for the organization issuing the billing record. Mandatory for EXTERNAL billing record if partyId is null.", example = "3456789123")
+	@ValidOrganizationNumber(nullable = true)
+	private String organizationNumber;
```

Comment:
According to Jakob, the recipient could be a private person as well. Maybe rename attribute to "legalId"?